paddle/pybind/pybind.cc

```diff
@@ -391,6 +392,13 @@ All parameter, weight, gradient are variables in Paddle.
              self.set_falsenet(net.Clone());
            });
 
+  py::class_<framework::Executor>(m, "Executor")
+      .def(py::init<std::vector<platform::Place> &>())
+      .def("run", [](Executor &self, const ProgramDesc &program_desc, int i) {
```

Comment:
can not see the meaning of `i`, maybe use `block_id` is better to read code

Reply:
Good idea!